api/app/signals/apps/questionnaires/models/question_graph.py

```diff
@@ -46,3 +49,32 @@ def get_trigger_order(self, question):
 
     def set_trigger_order(self, question, ids):
         return self._set_model_order(question, Trigger, ids)
+
+    @property
+    def reachable_questions(self):
+        from signals.apps.questionnaires.models.utils import retrieve_reachable_questions
+        return retrieve_reachable_questions(self)
+
+    def draw(self, location='/'):
+        from signals.apps.questionnaires.models.utils import draw_graph
+        return draw_graph(self, location=location)
+
+    def draw_base64(self):
+        """
+        Image as base64 encoded string
+        """
+        import tempfile
+        with tempfile.TemporaryDirectory() as tmp_dir:
```

Comment:
Two small comments here:
* as mentioned elsewhere I would rather be using a memory buffer throughout
* we should (not part of this PR) look into using a `tempfs` mounts for Docker (and also if and how Docker container disk usage changes when you create a temporary directory that is subsequently deleted)